services/frontend-v3/src/components/connections/ConnectionsView.jsx

```diff
@@ -2,14 +2,14 @@ import React from "react";
 import PropTypes from "prop-types";
 import { Row, Col, List } from "antd";
 
-const FriendsView = ({ friends }) => {
+const ConnectionsView = ({ Connections }) => {
```

Comment:
props should be camelCased
```suggestion
const ConnectionsView = ({ connections }) => {
```